plugin.yaml

```diff
@@ -160,6 +160,25 @@ node_types:
                     Defaults to GMT without daylight savings.
                 type: integer
                 default: 90
+            windows_organization:
+                description: >
+                    Organization name to use for sysprep customization.
+                type: string
+                default: Organization
+            use_custom_sysprep:
```

Comment:
why is it necessary to have a boolean property for indicating whether custom sysprep is used?
when custom_sysprep has a value, doesn't it mean the user provided a custom sysprep and wants to use it?